server_awsapigateway.html

```diff
@@ -0,0 +1,42 @@
+--- layout: default title: enable cross-origin resource sharing ---
+
+<div class="container">
+	<section>
+		<h1>CORS on AWS API Gateway</h1>
+		<p>
+			In your <a href="https://console.aws.amazon.com/apigateway/home">AWS API Gateway</a> console do the following:
+		</p>
```

Comment:
This first paragraph seems superfluous and can be merged with the second, e.g. remove this first paragraph and add a link to "Amazon API Gateway" in the next paragraph.